Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
Please fix all the queries in this workbook to make it specific to the CyberArk data connector. Currently it's just directly querying the CommonEventFormat (CEF) table which is shared by other CEF data connectors as well. Please add the following  after CommonEventFormat to each query:
"\n| where DeviceVendor == \"Cyber-Ark\" \n| where DeviceProduct == \"Vault\" \n|" to each of the queries in this workbook and please re-validate before re-submitting the workbook. 
I've given an example with one of the queries below. Please do the same for other queries in this workbook too. Thanks.